Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -363,12 +363,17 @@ ceph osd primary-affinity osd.0 0
 
 ## OSD Dedicated Network
 
+!!! outdated
```

Comment:
outdated is one of the rendered keywords? I'm not even sure where the list is defined.

Reply:
Good point. It looks like these are the ones we can use: https://squidfunk.github.io/mkdocs-material/reference/admonitions/

I think `info`, `tip` (more catching icon than info), or `warning` could be good choices.